src/PickerTop.js

```diff
@@ -0,0 +1,44 @@
+import React, {PropTypes} from 'react';
+import cx from 'classnames';
+
+export default React.createClass({
+  propTypes: {
+    fixed: PropTypes.bool,
+    handleClick: PropTypes.func,
+    nextDate: PropTypes.func,
+    previousDate: PropTypes.func,
+    title: PropTypes.string,
+    textClassNames: PropTypes.string,

```

Comment:
you're using `textClassNames` to style `title` which is a bit weird, I would change `title` in `value` and `textClassNames` in `valueClassName`